.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,33 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.trim.outputs.version }}
+    steps:
+      - id: trim
+        run: echo "::set-output name=version::${TAG:1}"
+        env:
+          TAG: ${{ github.event.release.tag_name }}
+
+  publish-npm:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Set version and publish
+        run: |
+          npm install -g json
+          json -I -f package.json -e "this.version=\"$VERSION\""
```

Comment:
As discussed on Slack, I'm not sure about this approach of not updating `package.json` in the repo, so it will not match the one deployed on npm. I think it's good to have a commit in master that bumps this package.json for each release, it helps to understand when a version has been made while browser the git history.

Reply:
Removed this part